config/karma.config.js

```diff
@@ -41,6 +41,7 @@ module.exports.getBaseKarmaConfig = function (opts = { ignoreBower: false }) {
 
 			// list of files / patterns to load in the browser
 			files: [
+				require.resolve('@babel/polyfill/dist/polyfill.js'),
```

Comment:
We probably shouldn't include this in our tests as then our tests won't catch when we have forgotten to define a required polyfill in `origami.json` -- I think those polyfills are included?

Reply:
We need to in order to get async functions working which seem to require the regenerator runtime. I couldn't get it to work any other way.